demos/http3/ossl-nghttp3.c

```diff
@@ -567,6 +567,7 @@ int OSSL_DEMO_H3_CONN_handle_events(OSSL_DEMO_H3_CONN *conn)
     int ec, fin;
     size_t i, num_vecs, written, total_written, total_len;
     int64_t stream_id;
+    uint64_t flags;
```

Comment:
Please initialize this variable here.

Remember, it's always a good practice to initialize a variable with default values at the time of declaration.

Reply:
Why? This is not used uninitialized.

Reply:
Agreed @t8m - we really don't want to require such a change, as it would lead to a lot of code churn. Note that none of the variables are assigned "default" values here, or in much of the code base.

Best practice would be to assign to each variable as it is declared, and have it live for as narrow a scope as possible.  That’s not possible with OpenSSL coding style (C90) unfortunately.

If variables are not initialised to a default value, some compilers can detect when they are used without prior assignment - this only has to be found by one compiler to be useful. Always assigning a default precludes any compiler from doing this. Further, a "default" value is pretty much guaranteed to be wrong some of the time. 

In summary, I’d prefer to keep to the general style here, which is not to set a default value for everything